share/completions/setfacl.fish

```diff
@@ -1,4 +1,4 @@
-set __fish_facl_spec_keywords default user group mask other
+set -lx __fish_facl_spec_keywords default user group mask other
```

Comment:
This thing is only used in the function right after it.

Just put it there, and not even as a variable:

```fish
for keyword in default user group mask other
```